src/codeu/chat/client/commandline/Chat.java

```diff
@@ -48,9 +48,13 @@ public Chat(Context context) {
   //
   public boolean handleCommand(String line) {
 
-    final Scanner tokens = new Scanner(line.trim());
-
-    final String command = tokens.hasNext() ? tokens.next() : "";
```

Comment:
Have you tried compiling the code with these changes? You have removed `tokens` variable which is still being used in line 74. Am I missing something?